concrete/controllers/backend/file.php

```diff
@@ -316,4 +319,4 @@ public function getJSON()
         $r->setFiles($files);
         $r->outputJSON();
     }
-}
+}
```

Comment:
Is there a reason you removed this newline?

Reply:
Nope. My editor did it and I didn't pay attention.
I added it back